SensorValidate/SensorValidate.cs

```diff
@@ -1,33 +1,30 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 
 namespace SensorValidate
 {
     public class SensorValidator
     {
-        public static bool _give_me_a_good_name(double value, double nextValue, double maxDelta) {
-            if(nextValue - value > maxDelta) {
-                return false;
-            }
-            return true;
+        public static bool CheckValueJumps(double value, double nextValue, double maxDelta) {
```

Comment:
CheckValueJumps  name denotes that it will return true if there is a jump ....but your code return false.!.. and caller is expected to negate return value...